<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>navClass</class>
 <widget class="QDialog" name="navClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nav</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="NavGraphicsView" name="gView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addPolyBut">
       <property name="text">
        <string>Add Poly</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileBut">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frameNum">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="frameSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Radius:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dimEdit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="coordText">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Coord: (0,0)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="indexText">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Index: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>reload</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NavGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>NavGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="nav.qrc"/>
 </resources>
 <connections/>
</ui>
